4.7.1-windowsservercore-1709/build/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM microsoft/dotnet-framework-build:4.7.1-1-windowsservercore-1709
```

Comment:
We've changed the default shell to PowerShell on the runtime images, for the sake of consistency we should change the default shell here as well

Reply:
The SHELL is set to PowerShell in the base dotnet-framework-build image therefore it is not necessary here.